pandas/core/window/common.py

```diff
@@ -274,3 +275,39 @@ def _prep_binary(arg1, arg2):
     Y = arg2 + 0 * arg1
 
     return X, Y
+
+
+def _check_min_periods(
```

Comment:
how does this interact with _meets_minimum_period?

Reply:
`_check_min_periods` picks the final value of `min_periods` (baked into the cython code currently for other aggs). 

`_meets_minimum_period` checks the current window to see `min_periods` is satisfied.